libcxx/test/libcxx/containers/views/views.span/debug.iterator-indexing.pass.cpp

```diff
@@ -20,77 +20,156 @@ struct Foo {
     int x;
 };
 
+template <typename Iter>
```

Comment:
I would suggest renaming those to `assert.XXXXXXXX.pass.cpp`, not sure why we used `debug.XXXXXXXXX.pass.cpp` since those are not specific to the debug mode.

Reply:
Done